### PR TITLE
Comments styling, deleting and author field

### PR DESCRIPTION
**Pull request by TsarN**
Make the comment section look pretty. Require author's name field in addition to comment text. Add links to delete individual comments.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/Comment.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
+package com.google.sps;
+
+public class Comment {
+    private final String author;
+    private final String text;
+    private final long timestamp;
```
Comment (DarthGandalf) [inline portfolio/src/main/java/com/google/sps/Comment.java:7]: in what units? why not java.time.Instant?

**Comment by TsarN** [comment]
Yeah, datastore assigns its own ids

File: portfolio/src/main/java/com/google/sps/Comment.java (hunk 1 above)
Comment (TsarN) [reply portfolio/src/main/java/com/google/sps/Comment.java:7]: Good question :) I'm not really familiar with java's stdlib, will look into java.time.Instant now.

Comment (DarthGandalf) [reply portfolio/src/main/java/com/google/sps/Comment.java:7]: PR was merged without resolving tis

Comment (TsarN) [reply portfolio/src/main/java/com/google/sps/Comment.java:7]: So approved doesn't necessarily mean "ready to merge"?

Comment (DarthGandalf) [reply portfolio/src/main/java/com/google/sps/Comment.java:7]: It means I believe you can fix the issue I pointed at without my reapproval
But alright, I'll stop approving too early

**Pull request merged**